docs/spec/openapi/public-docs.yaml

```diff
@@ -0,0 +1,5703 @@
+
+
+    It is provided under license [Attribution-NonCommercial-NoDerivatives 4.0
+    International](https://creativecommons.org/licenses/by-nc-nd/4.0/).
+  version: '0.1'
+  license:
+    name: Elastic License 2.0
+    url: 'https://www.elastic.co/licensing/elastic-license'
+  contact:
+    name: APM Team
+servers:
+  - url: /
+security:
+  - apiKeyAuth: []
+  - secretToken: []
+tags:
+  - name: agent config
+    description: APIs that query the APM Server for configuration changes.
+    x-displayName: APM agent configuration
+  - name: event intake
+    description: >-
+      The events intake API is the internal protocol that APM agents use to talk
+      to the APM Server.
+    x-displayName: APM event intake
+  - name: opentelemetry intake
+    description: >
+      The OpenTelemetry intake API uses the OpenTelemetry Protocol (OTLP) to
+      send traces, metrics, and logs to APM Server. OTLP is the default transfer
+      protocol for OpenTelemetry and is supported natively by APM Server. APM
+      Server supports two OTLP communication protocols on the same port:
+      OTLP/HTTP (protobuf) and OTLP/gRPC.
+    x-displayName: APM OpenTelemetry intake
+  - name: server info
+    description: APIs that query general APM Server information.
+    x-displayName: APM server information
+paths:
+  /:
+    get:
+      summary: Get general server information
+      description: >
+        This lightweight endpoint is useful as a server up/down health check.
+
+        To configure authenticated access to the APM server, the instructions at
+        [APM API
+        key](https://www.elastic.co/guide/en/observability/current/api-key.html)
+        or [APM Secret
+        Token](https://www.elastic.co/guide/en/observability/current/secret-token.html)
+        must be followed to configure the correct permissions for APM access.
+      operationId: getServerHealth
+      tags:
+        - server info
+      responses:
+        '200':
+          $ref: '#/components/responses/200_server_info'
+  /config/v1/agents:
+    get:
+      summary: Get agent configuration changes
+      description: >
+        To configure authenticated access to the APM server, the instructions at
+        [APM API
+        key](https://www.elastic.co/guide/en/observability/current/api-key.html)
+        or [APM Secret
+        Token](https://www.elastic.co/guide/en/observability/current/secret-token.html)
+        must be followed to configure the correct permissions for APM access.
+      operationId: getAgentConfig
+      tags:
+        - agent config
+      parameters:
+        - in: query
+          name: service.name
+          required: true
+          schema:
+            type: string
+        - in: query
+          name: service.environment
+          schema:
+            type: string
+      responses:
+        '200':
+          description: A successful response.
+          content:
+            application/json:
+              schema:
+                type: object
```

Comment:
The response body is missing here; should be the same as for making a `post` request to this endpoint.